test/integration/api/ps.bats

```diff
@@ -44,3 +44,61 @@ function teardown() {
 	[ "${#lines[@]}" -eq  2 ]
 	[[ "${lines[1]}" == *"false"* ]]
 }
+
+@test "docker ps --filter" {
+	start_docker_with_busybox 2
+	swarm_manage
+
+	# Running
+	firstID=$(docker_swarm run -d --name name1 --label "match=me" --label "second=tag" busybox sleep 10000)
```

Comment:
Yeah, actually I've been discovering a few things that may lead to best practices.

In bats, by default, every command that doesn't exit with status 0 will fail automatically. As an additional benefit, we get its output on the logs.

Unless we expect a command to fail or we want to get its `stderr`, it's better to **not** use `run` I believe.

So I believe that instead of:
```
run foo
[ $status -eq 0 ]
```

We should always do
```
foo
```

And instead of
```
run foo
[ $status -eq 0 ]
local thing=$output
```

```
thing=$(foo)
```

If we want to get `foo` stdout/stderr, do pattern matching on `$output`, or check `$status` for something different than `0`, then `run` is the way to go.